src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMutation.cs

```diff
@@ -15,10 +15,12 @@
 namespace Microsoft.PowerFx.Core.Texl.Builtins
 {
     // Patch(dataSource:*[], Record, Updates1, Updates2,…)
-    internal class PatchImpl : PatchFunction, IAsyncTexlFunction3
+    internal class PatchImpl : PatchFunction, IFunctionInvoker
     {
-        public async Task<FormulaValue> InvokeAsync(FormulaType irContext, FormulaValue[] args, CancellationToken cancellationToken)
+        public async Task<FormulaValue> InvokeAsync(FunctionInvokeInfo invokeInfo, CancellationToken cancellationToken)
```

Comment:
>cancellationToken [](http://example.com/codeflow?start=101&length=17)

unrelated to this PR but we should test if we shall throw before calling PatchAsync at line 48 and AppendAsync at line 53

Reply:
no harm, but low value. The caller has already checked this, and there's no looping or network calls here.